components/Tabs/src/TabBarView/MDCTabBarView.h

```diff
@@ -34,6 +34,11 @@ __attribute__((objc_subclassing_restricted)) @interface MDCTabBarView : UIScroll
 /** The color of the Tab bar's background. */
 @property(nullable, nonatomic, copy) UIColor *barTintColor;
 
+/**
+ The color for the Ripple effect for touch feedback. If no value is provided, uses a default value.
+ */
+@property(nullable, nonatomic, copy) UIColor *rippleColor;
```

Comment:
How do you feel about this being _null_resettable_?

Reply:
I don't like exposing the default value. None of the other APIs on this class expose their defaults, but simply do "something". In this case I'm defaulting to the RippleView default value.

Reply:
Is that OK with you? If we keep it `nullable` for now and leave the default as an implementation detail?

Reply:
For me it's just confusing that if a client sets `rippleColor = nil` then a ripple still occurs and the ask what the `rippleColor` is we return `nil`. I can be convinced otherwise if you feel strongly about it.

Reply:
Ok. I'll open an issue to change all the other APIs.

Reply:
Opened #7787 to change all APIs.

Reply:
Changed this to `nonnull`.